### PR TITLE
adding details of db collections reqd by service

### PR DESCRIPTION
**Pull request by PhilipGlover**
add this to README under 'Building and Running Locally':

Screenshot 2020-10-13 at 10 17 16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,16 @@ Building and Running Locally
...
+1. Send a GET request using your REST client to /efs-submission-api/healthcheck. The response should be 200 OK with status=UP.
+1. A database named `efs_submissions` and the following collections are required:
+
+Collection name|Description|Data
+--------------------|---------------|------|
+submission|id, dates, presenter, form & file details|this and the database will be created by the service upon starting a submission, then populated as the user enters their data|
+category_templates|the 'groupings' for the forms|reference data manually populated from:<br>`src/main/resources/category_templates.json`|
+form_templates|the form ids, names etc.|reference data manually populated from:<br>`src/main/resources/form_templates.json`|
+company_auth_allow_list|emails of IP's who are allowed to see and submit Insolvency forms|manually populated e.g.<br> `{"emailAddress": "demo@ch.gov.uk"}`|
+payment_charges|payment templates required by payment service|reference data manually populated from:<br>`src/main/resources/payments-charges.json`|
```
Comment (hepsimo) [inline README.md:39]: Note: file name has changed to match collection, now `payment_templates.json`

**Review by hepsimo — CHANGES_REQUESTED** [review]
Please update filename to `payment_templates.json`

File: README.md (hunk 1 above)
Comment (PhilipGlover) [reply README.md]: done..  I should have remembered that!

**Review by hepsimo — APPROVED** [review]
Changes look good

**Pull request merged**